saw-remote-api/saw-remote-api/Main.hs

```diff
@@ -101,4 +106,12 @@ sawMethods =
      "SAW/set option"
      setOptionDescr
      setOption
+  , Argo.notification
+     "SAW/clear state"
+     clearStateDescr
+     clearState
+  , Argo.notification
+     "SAW/clear all states"
+     clearAllStatesDescr
+     clearAllStates
```

Comment:
Are these new pieces of functionality documented in `SAW.rst`?

Reply:
No, and if this is the thing we can automatically get the server to spit out (which I believe it is), we should look at either getting it automatically updated or have CI check that it is in sync with the implementation with PRs.

Reply:
I've added a [check_docs.sh](https://github.com/GaloisInc/saw-script/pull/1118/files#diff-2d5ac64e727201bde6c263ad6d47556f155c2cf504f73557eebe7b5dd0153b2f) script that checks that `saw-remote-api/docs/SAW.rst` matches the documentation generated by the server's `doc` command as a part of our CI. At least now the two won't drift apart without us knowing.